### DISCUSSION TITLE
Observation

### DISCUSSION THREAD

**Discussion by dt-r** [hl7au/au-fhir-core#28] [AU Core R1 Resource Priority List - CLOSED]
Upvote for inclusion of a Resource Profile of [Observation](http://hl7.org/fhir/R4/observation.html) Resource in AU Core Release 1.

Upvote each of the options in the comments below and comment to suggest an additional profile of Observation for inclusion in AU Core. 

Use thumbs down emoji to _down vote_ inclusion in AU Core.

Rule of thumb for inclusion in AU Core is that the Observation profile should be applicable across multiple use cases e.g. be relevant for prescribing, patient summary, diagnostic ordering, and diagnostic reporting.

**Comment by dt-r** [comment]
[AU Core Observation](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-observation.html)

**Comment by dt-r** [comment]
AU Profiles of Vital signs Observations

- [AU Core Waist Circumference](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-waistcircum.html)
- [AU Core Body Weight](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-bodyweight.html)
- [AU Core Blood Pressure](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-bloodpressure.html)
- [AU Core Vitals Panel](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-vitalspanel.html)
- [AU Core Body Temperature](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-bodytemp.html)
- [AU Core Respiration Rate](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-resprate.html)
- [AU Core Heart Rate](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-heartrate.html)
- [AU Core Body Height](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-bodyheight.html)
- [AU Core Head Circumference](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-headcircum.html)
- [AU Core BMI](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-bmi.html)
- [AU Core Oxygen Saturation](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-oxygensat.html)

**Comment by dt-r** [comment]
[AU Core Biological Sex Assigned at Birth](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-sexassignedatbirth.html)

**Comment by dt-r** [comment]
[AU Core Alcohol Status](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-alcoholstatus.html)

**Reply by rachelhayhurst** [reply]
How do alcohol screening scores like AUDIT-C relate to this?

**Comment by dt-r** [comment]
[AU Core Smoking Cease Date](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-smokingceasedate.html)
[AU Core Smoking Start Date](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-smokingstartdate.html)
[AU Core Smoking Status](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-smokingstatus.html)

**Comment by dt-r** [comment]
[AU Core Gestational Age](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-gestationalage.html)
[AU Core Estimated Date of Delivery](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-estimateddateofdelivery.html)
[AU Core Pregnancy Status](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-pregnancystatus.html)
[AU Core Gravidity](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-gravidity.html)
[AU Core Last Menstrual Period](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-lastmenstrualperiod.html)
[AU Core Parity](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-parity.html)

**Comment by dt-r** [comment]
[AU Core Diagnostic Result Observation](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-diagnosticresult.html)
[AU Core Diagnostic Imaging Result Observation](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-diagnosticresult-imag.html)
[AU Core Pathology Result Observation](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-diagnosticresult-path.html)
[AU Core Lipid Result](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-lipid-result.html)

**Comment by dt-r** [comment]
[AU Core Assertion of No Relevant Finding](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-norelevantfinding.html)

**Comment by dt-r** [comment]
[AU Core Health Program Participation Summary](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-healthprogramparticipation.html)

**Comment by davidmckillop-ADHA** [comment]
Refer to "[DiagnosticReport using a panel of Observations (as per EU) or not (as per US)](https://github.com/hl7au/au-fhir-core/discussions/22#discussioncomment-7399064)" in the DiagnosticReport thread.
